src/Parser/Mage/Fire/Modules/Items/MarqueeBindingsOfTheSunKing.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { formatMilliseconds, formatNumber } from 'common/format';
+import getDamageBonus from 'Parser/Mage/Shared/Modules/GetDamageBonus';
+
+const BONUS_DAMAGE = 3;
+const CAST_BUFFER = 250;
+
+const debug = false;
+
+class MarqueeBindingsOfTheSunKing extends Analyzer {
+
+  static dependencies = {
+		combatants: Combatants,
+	};
+
+  damage = 0;
+  beginCastTimestamp = 0;
+  castTimestamp = 0;
+  hasBindingsBuff = false;
```

Comment:
So far as I can tell, this flag is true iff the player has the bindings buff... so why not remove this flag entirely and just use `this.combatants.selected.hasBuff(...` instead?